src/index.html

```diff
@@ -17,6 +17,78 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo-moyo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
```

Comment:
Add empty lines between multiline sibling blocks of HTML for better readability. In this case, add an empty line before the `<nav>` tag to separate it from the `<header>` tag.